.env

```diff
@@ -0,0 +1,8 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgresql://postgres:postgres@localhost:5432/restaurantdb?schema=public"
+PORT=8080
```

Comment:
Please, this .env file must not be uploaded and the configurations shall be done in local env according to settings of each developer, add this file in a `.gitignore`

Reply:
the .env file inside a gitignore.